src/wp-admin/includes/class-wp-media-list-table.php

```diff
@@ -367,7 +367,7 @@ public function get_columns() {
 		if ( ! $this->detached ) {
 			$posts_columns['parent'] = _x( 'Uploaded to', 'column name' );
 
-			if ( post_type_supports( 'attachment', 'comments' ) ) {
+			if ( post_type_supports( 'attachment', 'comments' ) && ! empty( get_option( 'wp_attachment_pages_enabled' ) ) ) {
```

Comment:
Based on the usage in https://core.trac.wordpress.org/changeset/56657 for consistency, this can be simplified:


```suggestion
			if ( post_type_supports( 'attachment', 'comments' ) && get_option( 'wp_attachment_pages_enabled' ) ) {
```